### DISCUSSION TITLE
Desktop unresponsive / shortcut icons gone

### DISCUSSION THREAD

**Discussion by bertustenhoor** [BlackArch/blackarch#4108] [Support]
Type: Question

Hey everyone, I seem to have done something wrong somewhere.. I lost all my shortcut icons from my desktop and the desktop itself is unresponsive for either left- or rightclick. 

Until a few days ago everything was fine, I blamed the switching between multiple displays and 1 display for this. 

My OS version:
``NAME="BlackArch Linux"
ID=blackarch
PRETTY_NAME="BlackArch Linux"
ANSI_COLOR="0;34"
HOME_URL="https://www.blackarch.org/"
SUPPORT_URL="https://www.blackarch.org/"
BUG_REPORT_URL="https://www.blackarch.org/"

IMAGE_ID=blackarch-linux-slim
IMAGE_VERSION=2023.05.01
``
I would not know where to start debugging this. Any help?

Great getting me offtopic, but where is my question supposed to go then?

**Comment by noraj** [comment]
If it takes too long before getting an answer, you can join the community at https://matrix.to/#/#BlackArch:matrix.org.

**Reply by bertustenhoor** [reply]
It was a annoying problem, I re-installed the whole BA again. I gave up BA afterwards. Although I was very impressed with BlakArch

**Comment by saltasatelites** [comment]
Hi,
Have you solved this issue?
More info is necessary. What exactly were you doing or trying to do when this happened?
